src/encoded/commands/load_data.py

```diff
@@ -35,16 +37,19 @@ def main():
     #create db schema
     configure_dbsession(app)
 
+    env = app.registry.settings.get('env.name')
+
     load_test_data = app.registry.settings.get('snovault.load_test_data')
     print("****** load test data is %s" % (load_test_data))
     load_test_data = DottedNameResolver().resolve(load_test_data)
 
-    clear_tables=False
+    # clear database on mastertest when flag is set and on the right env
+    clear_tables = True if ('mastertest' in env and args.drop_db_on_mt) else False
+
+    # do not run on webprod/webprod2 unless we set --prod flag
+    if 'webprod' in env and not args.prod:
+        return
```

Comment:
You may want to print a warning here: "You must use the --prod flag when running this on webprod environment."